packages/tpetra/core/src/Tpetra_Details_unpackCrsGraphAndCombine_def.hpp

```diff
@@ -911,42 +878,89 @@ unpackCrsGraphAndCombine(
         importLIDs.getRawPtr(), importLIDs.size(),
         true, "import_lids");
 
-  auto local_graph = sourceGraph.getLocalGraph();
-  auto local_col_map = sourceGraph.getColMap()->getLocalMap();
+  // mfh Could fix the use of protected data by adding a method
+  // to CrsGraph that returns the "local, globally indexed graph".
+  // However, this function needs to change those arrays anyway,
+  // so we might as well make this function a friend of CrsGraph.
+
+  //FIXME (tjf 11 Oct 2018) need to use non-protected data
+  indices_type indices("indices", graph.k_gblInds1D_.extent(0));
```

Comment:
Wait, we should just fix the code or make this a `friend` function if we need to get to these data.  We shouldn't have to copy them here.  This function is an implementation detail of CrsGraph, so we should feel free to make it a friend of CrsGraph.  We already do that for other functions relating to pack and unpack.

Reply:
this function is already a friend of CrsGraph, just need to remove the comment

Reply:
Cool; thanks!